CHANGELOG.md

```diff
@@ -3,6 +3,21 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [0.8.1](https://github.com/chanzuckerberg/edu-design-system/compare/v0.8.0...v0.8.1) (2022-03-03)
```

Comment:
Wat. This is the first time  I noticed there's a top-level changelog.

Why would lerna/conventional-commits maintain this, when each package already has its own changelog? 🤔  Weird